walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -13,11 +13,54 @@
 // limitations under the License.
 
 package com.google.sps;
+import com.google.sps.FindMeetingQuery;
+import com.google.sps.MeetingRequest;
+import com.google.sps.TimeRange;
+import java.util.*;
 
-import java.util.Collection;
 
 public final class FindMeetingQuery {
```

Comment:
it would be cool to describe in words somewhere what the idea behind your algorithm was, what the running time should be, and and advantages/disadvantages of this approach. in code, this is often done as javadoc on the class, so that someone opening up this file to read it can quickly get a "briefing" from the developer before diving into the actual code. See com.google.sps.Event for an example of the format and https://docs.oracle.com/javase/8/docs/api/java/util/HashSet.html for an example of describing the algorithm (HashSet's documentation is probably more detailed than necessary for this case though!)